spec/game_teams_spec.rb

```diff
@@ -0,0 +1,121 @@
+require './lib/game_teams'
+require 'pry'
+
+RSpec.describe GameTeams do
+    describe '#initialize' do
+
+        before do
+            
+            @game_teams_info = { 
+                            game_id: 2012030221, team_id: 3, HoA: "away", result: "LOSS", 
+                            settled_in: "OT", head_coach: "John Tortorella", goals: 2, shots: 8, 
+                            tackles: 44, pim: 8, powerPlayOpportunities: 3, powerPlayGoals: 0, 
+                            faceOffWinPercentage: 44.8, giveaways: 8, takeaways: 17 
+                            }
+            
+            @game_teams = GameTeams.new(@game_teams_info)
+
+        end
+
+        it 'exists' do
+            
+            expect(@game_teams).to be_a(GameTeams)
+
+        end
+
+        it 'has a game_id' do
+
+            expect(@game_teams.game_id).to eq(2012030221)
+
+        end
+
+        it 'has a team_id' do
+            
+            expect(@game_teams.team_id).to eq(3)
+
+        end
+
+        it 'can return if a team is home or away' do
+
+            expect(@game_teams.hoa).to eq("away")
+
+        end
+
+        it 'can return game result' do
+
+            expect(@game_teams.result).to eq("LOSS")
+
+        end
+
+        it 'can return how the game was settled' do
+
+            expect(@game_teams.settled_in).to eq("OT")
+
+        end
+
+        it 'returns a teams head coach' do
+
+            expect(@game_teams.head_coach).to eq("John Tortorella")
+
+        end
+
+        it 'returns goals by specific team in a game' do
+
+            expect(@game_teams.goals).to eq(2)
+
+        end
+
+        it 'returns how many shots a team attempted' do
+
+            expect(@game_teams.shots).to eq(8)
+
+        end
+
+        it 'returns how many tackles a team made' do
+
+            expect(@game_teams.tackles).to eq(44)
+
+        end
+
+        it 'returns pim aka penulty minutes for a team' do
+
+            expect(@game_teams.pim).to eq(8)
+        
+        end
+
+        it 'returns power play opportunities for a team' do
+            #should i keep the method name consistent with the hash key, or change the hash key
```

Comment:
Good question. Ruby convention would tell us to make it into something different than what we're seeing in our csv file if it doesn't fit that camel_case convention that we should be using. So you should change this one to power_play_opportunities or power_play_opps like you mentioned in the comment.

Reply:
modified method names to be snake case power_play_opps, power_play_goals, and faceoff_win_percent